testcases.py

```diff
@@ -770,6 +770,9 @@ def check(self) -> TestResult:
             direction = get_direction(p)
             packet_type = get_packet_type(p)
             packet_size = int(p.udp.length)
+            if packet_type == PacketType.VERSIONNEGOTIATION:
+                logging.info("Didn't expect a Version Negotiation packet.")
+                return TestResult.FAILED
```

Comment:
Why is this necessary? We don't check this for any of the other tests.

Reply:
The network simulator is dropping packets 2 - 7 sent from the client. In the case of version negotiation, the actual connection attempt wouldn't start before packet 8.